content/docs/languages/go/basics.md

```diff
@@ -388,9 +392,11 @@ with the server. We create this by passing the server address and port number to
 `grpc.Dial()` as follows:
 
 ```go
-conn, err := grpc.Dial(*serverAddr)
+var opts []grpc.DialOption
+...
+conn, err := grpc.Dial(*serverAddr, opts...)
```

Comment:
Like above, I added the `var opts` and the `opts` as an argument to better match the example code in the repo. Ok?

Reply:
Same answer.  Except it's always required on a client to specify `WithInsecure` or provide transport creds.